alpha/apps/kaltura/lib/kEntryPlayingDataHelper.php

```diff
@@ -0,0 +1,443 @@
+<?php
+class kEntryPlayingDataHelper
```

Comment:
I don't know why you need to split the logic between kContextDataHelper and kEntryPlayingDataHelper.